tests/e2e_tests/integration/08-rbac.spec.ts

```diff
@@ -51,60 +58,109 @@ test.describe('RBAC functionality', () => {
     await dialog.locator('text=Select​').nth(1).click({ force: true });
     await page.locator('text=Configure').click();
     await page.press('body', 'Escape');
-
-    await page.waitForTimeout(timeouts.oneSecond);
-    await dialog.locator('text=Search release tags​').click({ force: true });
-    await page.locator('li[role="option"]:has-text("All releases")').click({ force: true });
-    await dialog.locator('text=Select​').first().click({ force: true });
-    await page.locator('li[role="option"]:has-text("Read")').click();
-    await page.press('body', 'Escape');
-    await dialog.locator('text=Submit').scrollIntoViewIfNeeded();
-    await dialog.locator('text=Submit').click();
+    await dialog.getByRole('button', { name: /submit/i }).scrollIntoViewIfNeeded();
+    await dialog.getByRole('button', { name: /submit/i }).click();
   });
 
-  test('allows user creation', async ({ baseUrl, environment, loggedInPage: page, password, username }) => {
-    await page.goto(`${baseUrl}ui/settings/user-management`);
-    await page.getByRole('button', { name: /Create new user/i }).click();
-    await page.getByPlaceholder(/email/i).click();
-    await page.getByPlaceholder(/email/i).fill(`limited-${username}`);
-    await page.getByPlaceholder(/Password/i).click();
-    await page.getByPlaceholder(/Password/i).fill(password);
-    if (['enterprise', 'staging'].includes(environment)) {
-      await page.getByRole('combobox', { name: /admin/i }).click();
-      // first we need to deselect the default admin role
-      await page.getByRole('option', { name: 'Admin' }).click();
-      await page.getByRole('option', { name: 'testRole' }).click();
+  test('allows role creation for release tags', async ({ environment, loggedInPage: page }) => {
+    test.skip(!['enterprise', 'staging'].includes(environment));
```

Comment:
does that mean I will not be able to run this test without modifications here on my local setup?

Reply:
You should only need to pass a `--enterprise` to the run script or set `TEST_ENVIRONMENT=enterprise` if you want to run them with the playwright goodness against your local setup 👌
https://github.com/mendersoftware/gui/blob/master/.gitlab-ci.yml#L160